docs/intro-spoke.md

```diff
@@ -12,22 +12,24 @@ Spoke is an online 3D scene editor developed by the Hubs Community. With Spoke,
   <source src="img/architecture-kit.mp4" type="video/mp4">
 </video>
 
-### ✏️ Create
+## ✏️ Create
 
 No external software or 3D modeling experience required &mdash; build 3D scenes using the Spoke web editor so you can have a space that's entirely custom to your needs. From a board room to outer space and beyond, your space is in your control.
 
-### 🔭 Discover
+## 🔭 Discover
 
 Explore images, videos, and 3D models from around the web, all without opening up a new tab. With media integrations from Sketchfab and Google Poly, you'll be on your way to creating a scene in no time.
 
-### 🎉 Share
+## 🎉 Share
 
 Invite people to meet in your new space by publishing your content to Hubs immediately. With just a few clicks, you'll have a world of your own to experience and share &mdash; all from your browser.
 
 
 ![Screenshot of Spoke](img/intro-spoke-screenshot-min.jpeg)
 
 
-<!-- ## Getting started 
+## Getting started
 
-To build your first custom 3D environments check out Spoke's [interactive tutorial](https://hubs.mozilla.com/spoke/projects/tutorial) or take a look at the [Spoke documentation](./spoke-creating-projects.html). -->
+To build your first custom 3D environments take a look at the [Spoke documentation](./spoke-creating-projects.html).
+
+[//]: # (check out Spoke's [interactive tutorial]&#40;https://hubs.mozilla.com/spoke/projects/tutorial&#41; or )
```

Comment:
The interactive tutorial actually still works (provided you go to https://your-instance.domain/spoke/projects/tutorial).  Would it be possible to uncomment and explain this?  I know it won't be a clickable link, but it is a nice tutorial, and since you're uncommenting this section anyway....

Reply:
Done

Reply:
Excellent!  Merging.